lib/cartopy/mpl/gridliner.py

```diff
@@ -90,13 +97,42 @@ def _north_south_formatted(latitude, num_format='g'):
                                            _north_south_formatted(v))
 
 
+def _text_angle_to_specs_(angle):
+    """Get appropriate kwargs for a rotated label from its angle in degrees"""
+    if False and matplotlib.__version__ > '3':
+        # deactivated because does not work properly
+        # with rotation_mode='anchor'
```

Comment:
Can you please create an issue in matplotlib to address this, and also create an issue in cartopy referencing the matplotlib issue.  And can you then reference the cartopy issue in this comment (and vice versa) so that we can revert the deactivation here when appropriate.

Reply:
I simply updated the required mpl version to 3.1.